content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -140,6 +140,45 @@ using [TokenRequest](/docs/reference/kubernetes-api/authentication-resources/tok
 to obtain short-lived API access tokens is recommended instead.
 {{< /note >}}
 
+## Auto-generated legacy ServiceAccount token clean up
+
+Starting with v1.29, auto-generated legacy ServiceAccount tokens will be marked
+as invalid if they have not been used for a specified duration
+(defaults to one year). If the invalidated tokens have not been used for a
+specified duration (defaults to one year), they will be cleaned up by the
+control plane.
```

Comment:
How do I parse these ?
It takes one year for the token to be marked invalid.
It then takes another year for the token to be cleaned up.
Right?

Reply:
Yes. I revised again to make it more clear.